tests/run_tests.sh

```diff
@@ -305,6 +306,13 @@ function run_group_tests()
     python3 -m pytest ${TEST_CASES} ${PYTEST_COMMON_OPTS} ${TEST_LOGGING_OPTIONS} ${TEST_TOPOLOGY_OPTIONS} ${EXTRA_PARAMETERS} --cache-clear
 }
 
+declare -a flaky_tests=("test_bgp_slb.py" \
+                        "test_bgp_update_timer.py" \
+                        "test_everflow_testbed.py" \
+                        "test_vlan_ping.py" \
+                        "test_auto_techsupport.py" \
+                        "test_platform_info.py")
+
```

Comment:
Is it a good idea to hard code list of flaky tests here?

Reply:
Its been a while this is due to merge issue doesn't belong to this PR. I will have re-merge.